pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -118,3 +118,11 @@ svg {
 .pf-c-input-group :focus {
     z-index: 2;
 }
+
+// Alerts use elements that have fonts set in other frameworks (including PF3);
+// generally, this is an H4 that often has a font size and sometimes family set.
+// Therefore, it should inherit from the alert font set at the pf-c-alert level.
+// https://github.com/patternfly/patternfly/issues/4004
+.pf-c-alert__title {
+    font: inherit;
```

Comment:
I think you should put font-size: inherit; font-family: inherit
The font-weight is set propertly from the alert CSS and we need that.

With your changes the alerts will be:
![Screen Shot 2021-04-16 at 12 29 39](https://user-images.githubusercontent.com/14921356/114855458-47f1cd00-9de6-11eb-91ab-15b28716bc8e.png)

But they should be like this:
![Screen Shot 2021-04-16 at 12 29 28](https://user-images.githubusercontent.com/14921356/114855592-6e176d00-9de6-11eb-86b4-e7a6d167eea5.png)

Reply:
Huh. Weird. Font is shorthand for those and _should_ cover it, but being explicit is fine too. And you showed there is a difference in this case.

_Thanks for catching this!_

I guess this means it needs to be adjusted on the upstream issue too...